### PR TITLE
Exclude locale files from requiring a review from leads

### PR DESCRIPTION
**Pull request by differsthecat**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other
```

## Objective

Exclude locale files from requiring a review from leads

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -76,3 +76,9 @@ libs/components @bitwarden/team-platform-dev
 
 ## Desktop native module ##
 apps/desktop/desktop_native @bitwarden/team-platform-dev
+
+## Locales ## 
+apps/browser/src/_locales/en/messages.json
```
[the remark this answers is not in the document]
Comment (differsthecat) [reply .github/CODEOWNERS:81]: I'm not sure, my best guess is that someone at some point wanted it to be the top folder in browser's `src` folder?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -76,3 +76,9 @@ libs/components @bitwarden/team-platform-dev
 
 ## Desktop native module ##
 apps/desktop/desktop_native @bitwarden/team-platform-dev
+
+## Locales ##
```
Comment (withinfocus) [inline .github/CODEOWNERS:80]: 💭 I see https://github.com/bitwarden/clients/pull/5694 and it using section title for unowned files. Should we follow this?

Comment (differsthecat) [reply .github/CODEOWNERS:83]: I moved these under Multiple file owners

**Review by djsmith85 — APPROVED** [review]
Approving this. The error regarding the rename of team-client-integrations-dev is also present on master and will be addressed by them.

**Comment by withinfocus** [comment]
It's my understanding that `CODEOWNERS` won't work altogether when there's an error like this, so it's important we get that patched quickly.

**Comment by differsthecat** [comment]
> It's my understanding that `CODEOWNERS` won't work altogether when there's an error like this, so it's important we get that patched quickly.

Oops, I did not realize there was actually an error. I'll see about fixing it

**Pull request merged**